operator/v1/types_dns.go

```diff
@@ -95,6 +95,22 @@ type DNSSpec struct {
 	// +kubebuilder:default=Normal
 	OperatorLogLevel DNSLogLevel `json:"operatorLogLevel,omitempty"`
 
+	// nodeServices specifies a list of service objects for which host level resolvable entries should be added.
+	// Services in this list will be added to /etc/hosts on each node in the cluster by the node resolver.
+	// When not specified, only the default image registry service is resolvable.
+	// Services in this list will be added in addition to the default "image-registry.openshift-image-registry.svc" service.
+	// The default image registry service cannot be removed.
+	// For each service reference, entries will be created using the format "<name>.<namespace>.svc"
+	// and an alias with the CLUSTER_DOMAIN suffix of cluster.local will also be added.
```

Comment:
I suspect we probably want to shorten this and drop `CLUSTER_DOMAIN` if it is not configurable, CC @Miciah to confirm